ai-platform-unified/tutorials/unofficial/tf-keras-image-classification-custom-train-multi-replica/sdk_custom_training.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import argparse
+import logging
+
+from google.cloud import aiplatform
+
+logging.basicConfig(level=logging.INFO)
+
+def create_custom_training_job(
+    project,
+    staging_bucket,
+    location,
+    display_name,
+    script_path,
+    requirements,
+    container_image_uri,
+    container_args,
+    replica_count,
+    machine_type,
+    accelerator_type,
+    accelerator_count,
+):
+  aiplatform.init(project=project, staging_bucket=staging_bucket, location=location)
+  custom_training_job = aiplatform.CustomTrainingJob(
+      display_name=display_name,
+      container_uri=container_image_uri,
+      script_path=script_path,
+      requirements=requirements,
+  )
+
+  gcs_output_uri_prefix = f'gs://{staging_bucket}/{display_name}'
+
+  custom_training_job.run(
+      args=container_args,
+      base_output_dir=gcs_output_uri_prefix,
+      replica_count=replica_count,
+      machine_type=machine_type,
+      accelerator_type=accelerator_type,
+      accelerator_count=accelerator_count,
+  )
+
+  return custom_training_job, gcs_output_uri_prefix
+
+def parse_args():
+
+  parser = argparse.ArgumentParser(
+      description='Keras Image Classification using Multi Replica with Custom Script')
+  parser.add_argument(
+      '--project', default=os.getenv('PROJECT'), type=str,
+      help='Google Cloud project ID')
+  parser.add_argument(
+      '--location', default=os.getenv('LOCATION'), type=str,
+      help='Google Cloud project location')
+  parser.add_argument(
+      '--bucket', default=os.getenv('BUCKET'), type=str,
+      help='Google Cloud bucket used for staging')
+  parser.add_argument(
+      '--display-name', type=str, help='Training display name')
+  parser.add_argument(
+      '--container-image-uri', default=os.getenv('CONTAINER_IMAGE_URI'), type=str,
+      help='Google Container Registry URI of the training container image')
```

Comment:
Maybe:

"Artifact Registry URI of the pre-built training container image"

(since this is being used with a script, I believe it has to be one of the URIs on this page: https://cloud.google.com/vertex-ai/docs/predictions/pre-built-containers - you could even mention this explicitly in the help text if you want)